src/FluentCheck.ts

```diff
@@ -11,14 +11,16 @@ export class FluentResult {
   constructor(
     public readonly satisfiable = false,
     public example: FluentPicks = {},
-    public readonly seed?: number) { }
+    public readonly seed?: number,
+    public readonly withStatistics: boolean = false,
```

Comment:
What does showing test cases have to do with statistics?

Reply:
> What does showing test cases have to do with statistics?

My idea is that the flag will be used not only for showing test cases but also for all the more complex confidence improving aspects of the output like graphs and variable distributions that would increase the test execution time.

Reply:
Then call it `withTestCaseOutput` (or a better name) and when the user wants to turn on statistics (using `withStatistics`), the test case output collection is also turned on but without actually outputting in the end.

Reply:
To clarify:

`.withStatistics()` (should collect test cases but not output them)
`.withTestCaseOutputs()` (should collect test cases and output them)
`.withTestCaseOutputs().withStatistics()` (should collect test cases, output them and output statistics)

Reply:
I'm going to hijack this thread with some future ideas for this feature. For example: `.withTestCaseOutputs({onSuccess: false, maxTestCases: 100})`.